README.md

```diff
@@ -2,167 +2,47 @@
 
-```bash
-make install
-```
+## Usage
 
-- Uninstall CRDs to cluster in current kubeconfig context
+See the [user documentation](./docs/user/README.md).
 
-```bash
-make uninstall
-```
+## Installation
 
-- Run the operator locally (uses current kubeconfig context)
+See the [installation instruction](./docs/contributor/installation.md).
 
-```bash
-make run
-```
-
-- Build container image and deploy to cluster in current kubeconfig context
-
-```bash
-export IMG=<my container repo>
-make docker-build
-make docker-push
-kubectl create ns kyma-system
-make deploy
-```
-
-- Clean up everything
-
-```bash
-make undeploy
-```
-
-- Deploy module with Lifecycle Manager on a k3d cluster
-
-```bash
-make e2e-deploy-module
-```
-
-## Troubleshooting
-
-### Enable pausing reconciliations
-
-You must pause reconciliations to be able to debug the pipelines and, for example, try out a different pipeline configuration or a different OTel configuration. To pause reconciliations, create a `telemetry-override-config` in the operators Namespace.
-Here is an example of such a ConfigMap:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: telemetry-override-config
-data:
-  override-config: |
-    global:
-      logLevel: debug
-    tracing:
-      paused: true
-    logging:
-      paused: true
-    metrics:
-      paused: true
-```
-
-The `global`, `tracing`, `logging` and `metrics` fields are optional.
-
-#### Debugging steps
-
-1. Create an overriding `telemetry-override-config` ConfigMap.
-2. Perform debugging operations.
-3. Remove the created ConfigMap.
-4. To reset the debug actions, perform a restart of Telemetry Manager.
-
-   ```bash
-   kubectl rollout restart deployment telemetry-operator
-   ```
-
-**Caveats**
-If you change the pipeline CR when the reconciliation is paused, these changes will not be applied immediately but in a periodic reconciliation cycle of one hour. To reconcile earlier, restart Telemetry Manager.
-
-### Profiling
-
-Telemetry Manager has pprof-based profiling activated and exposed on port 6060. Use port-forwarding to access the pprof endpoint. You can find additional information in the Go [pprof package documentation](https://pkg.go.dev/net/http/pprof).
-
-### MackBook M1 users
+## Development
 
-For MacBook M1 users, some parts of the scripts may not work and they might see an error message like the following:
-`Error: unsupported platform OS_TYPE: Darwin, OS_ARCH: arm64; to mitigate this problem set variable KYMA with the absolute path to kyma-cli binary compatible with your operating system and architecture. Stop.`
+For details, see:
+- [Available commands for building/linting/installation](./docs/contributor/development.md)
+- [Testing Strategy](./docs/contributor/testing.md)
+- [Troubleshooting and Debugging](./docs/contributor/troubleshooting.md)
+- [Release process](./docs/contributor/releasing.md)
+- [Governance checks like linting](./docs/contributor/governance.md)
```

Comment:
@NHingerl @a-thaler How do we decide when to capitalize nouns? It looks somehow inconsistent 😉

Reply:
fixed